app/modules/training/datastore_training.html

```diff
@@ -0,0 +1,11 @@
+<html>
+  <body>
+    <h1>Datastore Training</h1>
+    <dl>
+      {% for entity in entities %}
```

Comment:
Let's add a conditional so `No entities found` is displayed if no entities exist.